client/src/components/Event/Event.tsx

```diff
@@ -88,7 +88,9 @@ export const Event = () => {
               </Stack>
               <Stack direction="row" spacing={1} alignItems="center">
                 <EventIcon />
-                <Typography variant="h6">{event?.timeAndDate}</Typography>
+                <Typography variant="h6">
+                  {new Date(event?.timeAndDate || 0).toDateString()}
```

Comment:
it is unnecessary to put 0 because it will put 1970, it will be better to not show anything